### ISSUE TITLE
Either schema for query-params

### ISSUE THREAD

**Issue by r0man** [Prismatic/fnhouse#29]
Hello,

I would like to use an either schema for query params in
fnhouse. Something like this:

```
(s/either
 {:top Double
  :right Double
  :bottom Double
  :left Double}
 {:latitude Double
  :longitude Double})
```

However, I can't compile the handlers because this schema doesn't
validate against `fnk-schema/InputSchema` over here:

https://github.com/Prismatic/fnhouse/blob/master/src/fnhouse/schemas.clj#L63

Would you accept an PR that changes the query params schema to
something like this:

```
(s/either schema.core.Either fnk-schema/InputSchema)
```

Is there a better way to do this? Or should
`fnk-schema/InputSchema` be extended to support `Either` in the
`plumbing` project?

Roman

**Comment by w01fe** [comment]
I don't see any reason not to change this to just `Schema` -- @davegolland what do you think?

**Comment by davegolland** [comment]
That's a good point. The current schema seems too restrictive, but making it be an arbitrary Schema maybe is too permissive? For instance, you can't have a sequence value passed in for the query params -- it must be a map with keyword keys. 

Also, the fnk-schema/InputSchema currently allows arbitrary schemas for the values, which also doesn't seem accurate for query params.

On second thought, I guess with coercion, anything is possible. So yeah, I'd be down for changing it to Schema.

**Comment by r0man** [comment]
Ok, cool. I submitted the PR https://github.com/Prismatic/fnhouse/issues/29

**Comment by davegolland** [comment]
Resolved with #30
